core/lib/merkle_tree/src/storage/rocksdb.rs

```diff
@@ -74,6 +74,15 @@ impl RocksDBWrapper {
         Ok(Self::from(RocksDB::new(path)?))
     }
 
+    /// Creates a new wrapper, initializing RocksDB with explicit options.
+    ///
+    /// # Errors
+    ///
+    /// Propagates RocksDB I/O errors.
+    pub fn with_options(path: &Path, options: RocksDBOptions) -> Result<Self, rocksdb::Error> {
```

Comment:
Bikeshedding: This constructor looks unnecessary, given that its implementation is fully public, so you could use it instead.

Reply:
OK